pkg/operator/configobservation/apienablement/observe_runtime_config.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var DefaultGroupVersionsByFeatureGate = map[configv1.FeatureGateName][]schema.GroupVersion{
-	"ValidatingAdmissionPolicy": {{Group: "admissionregistration.k8s.io", Version: "v1beta1"}},
+	"ValidatingAdmissionPolicy": {{Group: "admissionregistration.k8s.io", Version: "v1"}},
```

Comment:
You can remove this entirely, we don't need to pass `--runtime-config` because this group/version will be enabled by default.